fiftyone/utils/csv.py

```diff
@@ -78,8 +295,14 @@ class CSVDatasetExporter(foud.BatchDatasetExporter, foud.ExportPathsMixin):
             exported labels
         media_field ("filepath"): the name of the field containing the media to
             export for each sample
-        fields (None): a field or iterable of fields to include as columns in
-            the exported CSV file. By default, only the ``media_field`` is used
+        fields (None): an optional argument specifying the fields or
+            ``embedding.field.names`` to include as columns in the exported
+            CSV. Can be:
+
+            -   a field or iterable of fields
+            -   a dict mapping field names to column names
```

Comment:
Added support for passing `fields={"weather.label": "weather"}` to extract `Classification.label` values and store them in a `weather` column in the CSV file. See comment on unit tests for details.